build.ps1

```diff
@@ -0,0 +1,33 @@
+$pattern = "#pragma message:" 
+
+Write-Host "============> Build started."
+
+$buildOutput = &{
+    arduino compile --warnings default -b MIDICtrl20_MegaCommand:avr:mega .\sketch\
```

Comment:
Are you compiling here using the Arduino IDE?

I switched to Makefile because the Arduino IDE is extremely slow. 
Compiling source used to take around 5minutes. I can do it in under 20seconds with multi-threaded Make.

Reply:
I'm using the arduino CLI, it's much better than the IDE: https://github.com/arduino/arduino-cli